src/main/java/jp/ikedam/jenkins/plugins/groovy_label_assignment/GroovyLabelAssignmentAction.java

```diff
@@ -32,16 +33,39 @@
  */
 public class GroovyLabelAssignmentAction implements LabelAssignmentAction
 {
-    private Label label;
+    @Deprecated
+    transient private Label label;
+    
+    private final String labelString;

```

Comment:
It would be better to define the behavior in the case of null values in the code